django/researchdata/models.py

```diff
@@ -69,5 +70,23 @@ class Answer(models.Model):
     def __str__(self):
         return str(self.answer_text)[0:40]
 
+    def save(self, *args, **kwargs):
+        """
+        When a new model is saved email the research team
+        """
+
+        # Check if this is a new answer
+        if(self.meta_created_datetime is None):
+            # Send email alert to research team
+            send_mail(
+                        'CLiC Calendar: New Answer',
+                        'There has been a new answer submitted to CLiC Calendar.',
+                        'm.j.allaway@bham.ac.uk',
+                        ['m.j.allaway@bham.ac.uk'],
+                        fail_silently=False,
```

Comment:
What happens if it fails? I presume an error trying to send the email would abort the object saving - which may not be what you want.

Reply:
@edmondac I've wrapped this in a try except and printed any error. are you happy with this?

Reply:
Sure. I've switched print to logger in a set of suggestions.

Reply:
great, thanks, I've commited your suggested changes